src/others/contexts/dictionary-context.tsx

```diff
@@ -6,6 +6,8 @@ interface DictionaryState {
   locationDict: LocationsDict | undefined; // TODO: it's used only in map - doesn't have to be exposed once map is using the declarative API (translate...)
   translateLocation: (city_id: number) => Location | undefined;
   translateSupply: (category_id: string) => Supply | undefined;
+  isSuppliesLoaded: boolean;
```

Comment:
**issue**

I am not sure why do we need to check it, until it's loaded `suppliesDict` remains `undefined`. That's all we need no to know, isn't it?

Reply:
You're right. I can't remember why at the time I thought it was needed 🤔 . Removed and retested, and works as expected without.